### DISCUSSION TITLE
Create websocket server from node http server

### DISCUSSION THREAD

**Discussion by tuan3w** [uNetworking/uWebSockets.js#231] [Q&A] [answered]
Hi all,
I want to install additional http endpoints to websocket server. Socket.IO for examples, allows us to create socket.io instance from node http server. Can I do similar things in uWebsockets.js library ?
Thanks.

**Comment by tuan3w** [comment]
Sorry for not reading README carefully.

**Comment by sm64js** [comment]
Hey! I am wondering the same thing! I want to do this, I did read through the readme and the documentation, it is still unclear to me.  Can you help me out?

**Comment by ghost** [comment, marked as answer]
No you cannot do this because Node.js is entirely separate from uWS. That's why uWS comes with basic HTTP support to handle most cases.
